src/util/TokenService.js

```diff
@@ -48,7 +48,7 @@ class TokenService {
      * @param {string} tokenOrTokenFunction - Optional token or token function
      * @return {Promise} that resolves to a token
      */
-    static getToken(id: string, tokenOrTokenFunction: Token): Promise<?string> {
+    static async getToken(id: string, tokenOrTokenFunction: Token): Promise<?string> {
```

Comment:
Doesn't look like this function accepts an array of IDs anymore, can you update the jsdoc?

Reply:
fixed